themes/frontierline-nightly/content-views/content.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Display a full post, no featured image.
+ */
+?>
+
+<article id="post-<?php the_ID(); ?>" <?php post_class('post'); ?>>
+  <header class="entry-header">
+    <div class="entry-tools">
+    <?php if (has_category()) : ?>
+      <div class="categories">
+        <b><?php _e('Categories:', 'frontierline'); ?></b>
+        <?php the_category(' ') ?>
+      </div>
+    <?php endif; ?>
+
+    <?php if ((is_single() && get_option('frontierline_share_posts') == 1) || (is_page() && get_option('frontierline_share_pages') == 1)) :
+        get_template_part('includes/social-share');
+       endif; ?>
+    </div>
+
+    <h1 class="entry-title">
+    <?php if (!is_singular()) : ?>
+      <a href="<?php the_permalink(); ?>" title="<?php printf(esc_attr__('Permanent link to “%s”', 'frontierline'), the_title_attribute('echo=0')); ?>" rel="bookmark">
+        <?php the_title(); ?>
+      </a>
+    <?php else : ?>
+      <?php the_title(); ?>
+    <?php endif; ?>
+    </h1>
+
+    <div class="entry-info">
+    <?php if ($post->post_type === 'post') : ?>
+      <address class="vcard">
```

Comment:
The vcard is now configurable in the parent theme (`frontierline_no_byline` option). We should add it here too.